package.json

```diff
@@ -92,5 +92,6 @@
     "dns-over-http-resolver": "^2.1.1",
     "loupe": "^2.3.6",
     "vite": "^5.2.11"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha256.c17d3797fb9a9115bf375e31bfd30058cac6bc9c3b8807a3d8cb2094794b51ca"
```

Comment:
would put this below the `engines` field for better visibility

and do we need the commit hash? I don't see how this provides any value here

Reply:
This is what `corepack enable` does, looks reasonable to me. Probably used as a security measure.

Reply:
> Probably used as a security measure.

This wouldn't make sense because the user has yarn already installed, and it's a global package. But we can keep it of course

Reply:
Corepack changes things here

Reply:
let still move this up in the package.json file